src/main/java/com/gojek/beast/worker/BqQueueWorker.java

```diff
@@ -69,7 +71,7 @@ private Status pushToSink(Records poll) {
         if (status.isSuccess()) {
             boolean ackStatus = acknowledger.acknowledge(poll.getPartitionsCommitOffset());
             if (ackStatus) {
-                statsClient.timeIt("batch.processing.latency.time,size=" + poll.getRecords().size(), poll.getPolledTime());
```

Comment:
Please add one metric for the time taken to acknowledge the batch since it contributes to the end to end latency

Reply:
It is included.